lib/services/nodes-api-service.js

```diff
@@ -509,6 +509,33 @@ function nodeApiServiceFactory(
     };
 
     /**
+     * Remove the tag from a list of nodes
+     * @param  {String}     tagName
+     * @return {Promise}
+     */
+    NodeApiService.prototype.masterDelTagById = function(tagName) {
+        return Promise.resolve().then(function() {
+                assert.string(tagName, 'tag must be a string');
+        })
+        .then(function() {
+            return waterline.nodes.findByTag(tagName);
+        })
+        .then(function(nodes) {
```

Comment:
This could also be written as:
.map(function(node) {
  return node.remTags(node.id, tagName).then(function() { return node.id; });
});